### PR TITLE
Storage partitioning is beneficial to user privacy and security.

### PR DESCRIPTION
**Pull request by hober**
Here's my attempt at addressing #4, #7, and #8.

@arturjanc, @michaelkleber, @plinss: could you take a look?

### CODE REVIEW

**Comment by arturjanc** [comment]
Thanks @hober, this summary looks reasonable to me. Personally, I'd emphasize the risks a bit more (arguably, the two problems you listed in the _Benefits and Costs of Storage Partitioning_ section undermine core security/privacy properties we need to uphold on the web), but just capturing the concerns is already a solid improvement.

**Comment by michaelkleber** [comment]
Thanks Tess, fixes #7

**Review by LeaVerou — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -41,14 +69,15 @@
 <section id="abstract">
 
 This finding examines the sustainability of bundling
-and double-keyed caching in web development.
+and storage partitioning in web development.
```
Comment (ylafon) [inline index.html:72]: Storage partitioning is one aspect of double-keyed caching, see for example "Partitioning Network Cache"

Comment (hober) [reply index.html:72]: The network cache is a storage mechanism :)

File: index.html [hunk 2]
```diff
@@ -82,20 +111,22 @@
...
-This further amplifies the inefficiencies of transport.
-
-The end-user impact is unnecessary repeated downloads of resources
-which can mean delay, energy consumption, storage bloat, and data cost.
+Additionally,
+various privacy and security issues identified with client-side storage mechanisms
+have been mitigated by the introduction of storage partitioning,
+also known as double-keying. [[PARTITIONING]] [[COSI]] [[TIMING]] [[XS-LEAKS]]
+While necessary,
+this has had the regrettable effect
```
Comment (cynthia) [inline index.html]:
```suggestion
this has had a negative side effect
```

File: index.html [hunk 3]
```diff
@@ -82,20 +111,22 @@
...
+of amplifying transport inefficiencies.
+The repeated downloads of resources
+can mean delay, energy consumption, storage bloat, and data cost
```
Comment (cynthia) [inline index.html]:
```suggestion
can mean increased delay, energy consumption, storage bloat, and data cost
```

File: index.html [hunk 4]
```diff
@@ -108,26 +139,33 @@
...
-- Additional storage space to store these cached resources.
+- Lessened benefit of different websites loading shared resources from
+  a <abbr title="Content Delivery Network">CDN</abbr>,
+  which is not well understood by developers.
+- Additional storage space is required to store these cached resources.
 - Additional packages are transferred over the wire,
   which, in many scenarios, can be an actual financial burden for users.
 - Additional energy is consumed as extra CPU cycles are used for decoding, parsing, and executing resources.
-- Additional bandwidth that content users, intermediaries, and providers need to pay.
+- Additional bandwidth used must be paid for by server operators, users, and intermediaries.
```
Comment (cynthia) [inline index.html]:
```suggestion
- Additional bandwidth used must be paid for by users, intermediaries, and content providers.
```

File: index.html [hunk 5]
```diff
@@ -82,20 +111,22 @@
...
-
-The end-user impact is unnecessary repeated downloads of resources
-which can mean delay, energy consumption, storage bloat, and data cost.
+Additionally,
+various privacy and security issues identified with client-side storage mechanisms
+have been mitigated by the introduction of storage partitioning,
+also known as double-keying. [[PARTITIONING]] [[COSI]] [[TIMING]] [[XS-LEAKS]]
+While necessary,
+this has had the regrettable effect
+of amplifying transport inefficiencies.
```
Comment (cynthia) [inline index.html]:
```suggestion
of increased transport costs.
```

File: index.html (hunk 2 above)
Comment (hober) [inline index.html]:
```suggestion
this has had the negative effect
```

**Review by cynthia — APPROVED** [review]

**Pull request merged**